selfdrive/controls/lib/alerts.py

```diff
@@ -432,7 +432,7 @@ def __gt__(self, alert2):
       "openpilot Canceled",
       "Speed too low",
       AlertStatus.normal, AlertSize.mid,
-      Priority.HIGH, VisualAlert.none, AudibleAlert.chimeDisengage, .4, 2., 3.),
+      Priority.HIGH, VisualAlert.steerRequired, AudibleAlert.chimeWarningRepeat, 1., 3., 4.),
```

Comment:
mmm... but an audible alert (the same used for normal disengagement) is already issued. Your change just makes it louder, which I think is too annoying.
Sorry, I was under the impression that before your change OP wasn't issuing any audible alert.